src/dodal/devices/bimorph_mirrors/CAENels_bimorph_mirror_interface.py

```diff
@@ -0,0 +1,273 @@
+import time
+from enum import Enum, IntEnum
+from functools import partial
+from typing import Any, Union
+
+from bluesky.protocols import Movable
+from ophyd import Component, Device, EpicsSignal, EpicsSignalRO
+from ophyd.status import StatusBase, SubscriptionStatus
+
+from dodal.devices.status import await_value
+
+
+class ChannelAttribute(Enum):
+    VTRGT = ("VTRGT",)
+    VTRGT_RBV = ("VTRGT_RBV",)
+    SHIFT = ("SHIFT",)
+    VOUT = ("VOUT",)
+    VOUT_RBV = ("VOUT_RBV",)
+    STATUS = "STATUS"
+
+
+class OnOff(IntEnum):
+    ON = (1,)
+    OFF = 0
+
+
+class OperationMode(IntEnum):
+    HI = (0,)
+    NORMAL = (1,)
+    FAST = 2
+
+
+class Status(IntEnum):
```

Comment:
Potentially confusing to have a Status class when you're also using ophyd Statuses